frontend/src/services/bikesApiService.ts

```diff
@@ -1,16 +1,24 @@
 import axios from 'axios'
 
-const apiUrlSlug = '/api/bikes'
+const apiUrlSlug = '/api/bikes/'
 
 async function get() {
-   return await axios.get(apiUrlSlug).then((response) => response.data)
+   return await axios
+      .get(apiUrlSlug)
+      .then((response) => response.data)
+      .catch((error) => {
+         console.error(error)
+      })
 }
 
-
 async function post(newBikeTitle: string) {
-   return await axios.post(apiUrlSlug, {title: newBikeTitle})
+   return await axios
+      .post(apiUrlSlug, { title: newBikeTitle })
+      .then((response) => response.data)
+      .catch((error) => console.error(error))
```

Comment:
Catch error where method is called.

Reply:
The axios method is called here indeed. Here, I suggest, we want to catch a possible error. Later then, in the bikeApi-hook we want only to call the apiService-method disregarding it's actual return value. It is my opinion, I'm ready to discuss this. :)